src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -43,7 +55,16 @@ export const BasicDateInput = (props) => {
             inputFormat={format}
             minDate={dateProps.minDate}
             maxDate={dateProps.maxDate}
-            renderInput={(params) => <TextField variant="outlined" sx={{ flexGrow: 1 }} size="small" {...params} />}
+            renderInput={(params) => (
+              <TextField
+                variant="outlined"
+                sx={{ flexGrow: 1 }}
+                size="small"
+                error={!!errorType}
+                helperText={errorMessages[errorType] ?? ''}
```

Comment:
Adding an optional chaining here could help prevent some errors and crashes: `errorMessages?.[errorType] ?? ''`
(idem in the number input component)